workflows/charts/tgi/README.md

```diff
@@ -21,8 +21,9 @@ For more information about how to use Huggingface text-generation-inference with
 | fullnameOverride | string | `""` | Full qualified Domain Name |
 | ingress | object | `{"annotations":{},"className":"","enabled":false,"hosts":[{"host":"chart-example.local","paths":[{"path":"/","pathType":"ImplementationSpecific"}]}],"tls":[]}` | Ingress configuration |
 | nameOverride | string | `""` | Name of the serving service |
-| pvc.size | string | `"15Gi"` |  |
-| pvc.storageClassName | string | `"nil"` |  |
+| pvc | object | `{"create":true,"name":"model-server-cache","size":"15Gi","storageClassName":"nil"}` | Configure Storage Volume |
+| pvc.create | bool | `true` | Create a new PVC |
+| pvc.name | string | `"model-server-cache"` | Specify the name of either an existing or new PVC |
```

Comment:
pvc.size and pvc.storageClassName should be kept since we're adding entries for other vars in the pvc object as well.

Reply:
These were removed from the README because I shuffled the comments around. The configuration can be found in the new line 24.

The ability to modify the SCN is still easy to do, just want to make the table shorter.